src/styles/index.scss

```diff
@@ -1,3 +1,65 @@
+$day-size: 100px;
+$transition-duration: 0.5s;
+$hover-transform: -20px;
+$hover-background: #ffbfcb;
+$gap: 1px;
+
 body {
   margin: 0;
 }
+
+.calendar {
```

Comment:
Consider moving calendar BEM block to a separate file (according to BEM rules)
Same for variables